net/ser2net/Makefile

```diff
@@ -24,11 +24,16 @@ PKG_INSTALL:=1
 
 include $(INCLUDE_DIR)/package.mk
 
+CONFIGURE_ARGS += \
+	--with-pthreads \
+	--with-sysfs-led-support
+
 define Package/ser2net
   SECTION:=net
   CATEGORY:=Network
   TITLE:=Serial to Network Proxy
   URL:=https://sourceforge.net/projects/ser2net/
+  DEPENDS:=+libgensio +libyaml +libpthread
```

Comment:
It seems you are doing more changes than just "update to version 4.3.5" as said in the commit message.

Reply:
The updated version has updated requirements. It's still just an update.

Reply:
In that case, you need to describe those changes in the commit description.

Reply:
I changed the commit description to mention the new dependencies and the CONFIGURE_ARGS. Are the changes sufficient, or is there still something else?